src/media/js/views/homepage.js

```diff
@@ -59,6 +59,19 @@ define('views/homepage',
 
         if (isDesktop) {
             mktCarousel.initialize();
+            var promoLinks = {
```

Comment:
this seems like it should be exposed via the carousel module. I'd even purpose the mktCarousel to just be called desktopPromo since i doubt anyone's gonna want more than one carousel